src/Symfony/Component/AssetMapper/MappedAsset.php

```diff
@@ -120,9 +120,9 @@ public function setContent(string $content): void
         $this->content = $content;
     }
 
-    public function addDependency(self $asset, bool $isLazy = false): void
+    public function addDependency(self $asset, bool $isLazy = false, bool $contentDependency = false): void
```

Comment:
this method needs to have phpdoc describing those 2 arguments, as they are the ones that implementors of asset compilers will see in their IDE when using the code, not the AssetDependency constructor (alternatively, we might change this to `->addDependency(AssetDependency $dependency)` to avoid having to duplicate the signature)

Reply:
Good idea just to use `->addDependency(AssetDependency $dependency)`